test/unit/ItemValueTests/checkModularity.js

```diff
@@ -0,0 +1,34 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("calls the modular function the correct number of times", function () {
+            var item = mocks.mockItemValue(mocks.mockItemsHoldr(), "weight", {
```

Comment:
For all of these, can you add explicit `// Arrange`/`// Act`/`// Assert` comments?